### DISCUSSION TITLE
Is installation without cert-manager still available?

### DISCUSSION THREAD

**Discussion by kmlevy** [actions/actions-runner-controller#2163] [Questions] [answered]
I am using ARC v0.25 (chart 0.20) and am looking to upgrade to a newer version.  I notice that in the new documentation, there is no longer any documentation for running ARC w/out cert-manager.  Is that still an option for newer releases?

If it is still supported, can you point me to the current documentation?  The documentation I used was this (from specific version commit hash):  https://github.com/actions/actions-runner-controller/tree/edbdef8d20f6faad5f5db728826a6a3fc97a29be#using-without-cert-manager

I noticed that the documentation was recently re-worked, could this have been something that was missed?

**Comment by kevholmes** [comment]
@kmlevy there's a lot in-flux right now with this project after GitHub has taken ownership. It looks like this cert-manager requirement is actually going to be dropped entirely in an upcoming release: https://github.com/actions/actions-runner-controller/pull/2153. I'm pretty sure when 0.26.0 and chart 0.21.0 came out the docs still had references to the option for installing without cert-manager. I ran into some issues though (sorry I cannot remember what they were 😞) and ended up just using cert-manager..

**Reply by kmlevy** [reply]
Does that mean that the current version does not support it?  If it does support it, can we please get the documentation for it re-added?

**Comment by Link-** [comment, marked as answer]
The docs have not changed, information was moved around to be more accessible.

Setup without cert-manager is still documented, here: https://github.com/actions/actions-runner-controller/blob/master/docs/authenticating-to-the-github-api.md#using-without-cert-manager

Nothing was changed in the behaviour of ARC with regard to cert-manager in `v0.26.0` and `v0.27.0`. You can inspect the release notes for both versions for further details:

https://github.com/actions/actions-runner-controller/tree/master/docs/releasenotes

**Reply by Link-** [reply]
The behaviour of the new mode will have 0 impact on the existing modes and our plan is to have both of them work for the foreseeable future. We will publish separate documentation that covers the new mode in due time.
